roles/sys/repositories/tasks/os/Debian/edb-repositories.yml

```diff
@@ -2,6 +2,25 @@
 
 # © Copyright EnterpriseDB UK Limited 2015-2023 - All rights reserved.
 
+- name: Find existing EDB repositories
+  find:
+    paths: /etc/apt/sources.list.d/
+    patterns: 'edb-*.list'
+    file_type: "file"
+  register: existing_edb_repos
+
+- name: remove unwanted EDB repositories
```

Comment:
As with the yum version, here too I think we can just remove the files and run `apt-get update` afterwards if we removed >0 files.

Also, it might be worth using `rm …bunch of files we don't want…` instead of looping over `file`/`state: absent`. The latter will be much slower if there are actually multiple files.